packages/cmf/src/sagaRouter/index.md

```diff
@@ -0,0 +1,48 @@
+# Router Saga
+
+## Purpose
+
+To provide a way to start or stop saga, depending on current route of the application.
+
+Those saga can now provide some side effect based on route context.
+
+## Problems example that are solved.
+
+### Basic routing problems
+
+Lets say that our application handle shortcut navigation, "Escape" button triggering change of route.
```

Comment:
"Escape" button trigerring a `CANCEL_ACTION` to be dispatched wich should trigger a route change